aeron-driver/src/main/java/io/aeron/driver/Configuration.java

```diff
@@ -1096,4 +1098,68 @@ public static void validatePageSize(final int pageSize)
             throw new ConfigurationException("Page size not a power of 2: page size=" + pageSize);
         }
     }
+
+    private static int getSize(final String propertyKey, final int defaultValue)
+    {
+        final String propString = getProperty(propertyKey);
+        if (propString != null)
+        {
+            return parseSize(propertyKey, propString);
+        }
+        else
+        {
+            return defaultValue;
+        }
+    }
+
+    private static long getSize(final String propertyKey, final long defaultValue)
+    {
+        final String propertyValue = getProperty(propertyKey);
+        if (propertyValue != null)
+        {
+            return parseSize(propertyKey, propertyValue);
```

Comment:
The parseSize method below only supports int. Should do a long method with a range check for if the int value is allowed.